src/test-fbp/converter-error-expected.txt

```diff
@@ -0,0 +1 @@
+WRN: ./src/lib/common/sol-types.c:242 sol_irange_division() Division by zero: 10, 0
```

Comment:
Isn't this the case of sending a packet to the error port so it can be captured by a test node?